lib/application/pages/home/collections/update/update_collection_page.dart

```diff
@@ -174,3 +234,117 @@ class _MemosActionButton extends ConsumerWidget {
     );
   }
 }
+
+class _MemosReorderableList extends ConsumerWidget {
+  const _MemosReorderableList({
+    required this.memos,
+    required this.currentMemoIndex,
+    required this.onReorder,
+    required this.onPressed,
+  });
+
+  final List<MemoUpdateMetadata> memos;
+
+  /// Selected index of [memos].
+  ///
+  /// Indicates which memo is currently highlighted.
+  final int currentMemoIndex;
+
+  /// Called when a memo is moved from `oldIndex` to `newIndex`.
+  final void Function(int oldIndex, int newIndex) onReorder;
+
+  /// Called when a memo at `index` is pressed.
+  final void Function(int index) onPressed;
```

Comment:
> Not sure, even flutter use both nomenclatures in similar contexts, like InkWell.onTap and ElevatedButton.onPressed.

Yeah, their definition is painful (https://stackoverflow.com/a/61527357/8558606)

I would say for us to stick with the same strategy: `onPressed` for buttons (meaning ACTIONS) and `onTap` for overall interactable stuff.